src/transformers/onnx/__init__.py

```diff
@@ -15,4 +15,4 @@
 
 from .config import EXTERNAL_DATA_FORMAT_SIZE_LIMIT, OnnxConfig, OnnxConfigWithPast, PatchingSpec
 from .convert import export, validate_model_outputs
-from .utils import ParameterFormat, compute_serialized_parameters_size
+from .utils import ParameterFormat, compute_effective_axis_dimension, compute_serialized_parameters_size
```

Comment:
Is this import still needed as if I understand correctly it was previously used by the `generate_dummy_inputs` method?

Reply:
Yes, I forgot about that. I have removed it now. Thanks!